libffi/funcall.lisp

```diff
@@ -121,4 +120,6 @@
 
 (setf *foreign-structures-by-value* 'foreign-funcall-form/fsbv-with-libffi)
 
+(pushnew 'cffi-features:struct-by-value *features*)
```

Comment:
Hmm. cffi-features is obsolete and has been so for more than 10 years now. I think we can remove it! ;-) Maybe `:cffi-libffi` would be a good name? Why exactly do you need this feature anyway? Doesn't ASDF provide some nice way of detecting the presence or absence of some library?